website/templates/report.html

```diff
@@ -308,6 +309,53 @@ <h3 class="text-xl font-bold leading-none text-red-500">Latest Issues</h3>
                 </div>
                 <button type="button" onclick="validateCVE()">Validate</button>
             </div>
+            <div id="trademark-fields">
+                <div class="pt-5 mt-2">
+                    <label for="infringing_domain_name"
+                           class="block text-2xl font-semibold leading-6 text-gray-900">Infringing Domain Name</label>
+                    <input type="text"
+                           name="infringing_domain_name"
+                           id="infringing_domain_name"
+                           placeholder="Example.com"
```

Comment:
```suggestion
                           placeholder="example.com"
```

Reply:
resolved in https://github.com/OWASP-BLT/BLT/pull/2655/commits/3e76e5b033074d878415807aa0742ac40a9e7996